src/views/components/trace/trace-chart-table/trace-container.vue

```diff
@@ -21,7 +21,7 @@
         <div class="method">
           Method
         </div>
-        <div class="start-time">
+        <div class="start-time" lang="scss" style="width: 200px;">
```

Comment:
I mean the style should be here(https://github.com/apache/skywalking-rocketbot-ui/blob/master/src/views/components/trace/trace-chart-table/trace-item.vue#L59)

Reply:
> The pull request will update after you push codes, there is no need to reopen a new one.

ok

Reply:
You probably update `min-width` to resolve the problem without other change  https://github.com/apache/skywalking-rocketbot-ui/blob/master/src/views/components/trace/trace-chart-table/trace.scss#L26

Reply:
> You probably update `min-width` to resolve the problem without other change https://github.com/apache/skywalking-rocketbot-ui/blob/master/src/views/components/trace/trace-chart-table/trace.scss#L26

Yes,  This code of CSS is complex, Iam not good at CSS,  I think u can fix it